Dalamud/Game/SigScanner.cs

```diff
@@ -120,170 +126,166 @@ public sealed class SigScanner : IDisposable {
             }
         }
 
-        private IntPtr moduleCopyPtr;
-        private long moduleCopyOffset;
+        private IntPtr _moduleCopyPtr;
+        private long _moduleCopyOffset;
```

Comment:
Why the style change 🤔

Reply:
looks like their linter went wild and changed a bunch of random things, like the 4 spaces on all the comments lol

Reply:
wups yeah, think i still had it set to auto format/clean on paste